state/state.go

```diff
@@ -211,12 +211,38 @@ func ControllerAccess(st *State, tag names.Tag) (permission.UserAccess, error) {
 	return st.UserAccess(tag.(names.UserTag), st.controllerTag)
 }
 
-// RemoveAllModelDocs removes all documents from multi-model
+// RemoveModel removes all documents from multi-model
```

Comment:
The comment needs to be changed since RemoveModel() does more than just remove the documents